docs: add patch release notes for DOC-2957 (#10919)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

<ReleaseNotesVersions />

## July 4, 2026 - Release 4.9.x

<!-- PATCH RELEASE TICKET: DOC-2957 -->

### Improvements

<!-- https://spectrocloud.atlassian.net/browse/PE-8884 -->

> - The LocalTUI installation flow now detects Kairos partitions on all disks and blocks the install with a clear message

Reviewer: ```suggestion
- The Local TUI installation flow now detects Kairos partitions on all disks and blocks the install with a clear message
```

  identifying the affected disks, preventing unpredictable behavior from stale partitions.

<!-- https://spectrocloud.atlassian.net/browse/PE-8912 -->

- Custom UI wizard bundles can now be packaged via the [Palette CLI](../automation/palette-cli/palette-cli.md) and
  rendered in Local UI on Slim ISO-bootstrapped edge hosts, enabling content authors to ship tailored configuration
  experiences alongside their edge content.

### Bug Fixes

<!-- https://spectrocloud.atlassian.net/browse/PE-9004 -->

- Fixed an issue where Palette Agent upgrades were triggered on nodes in a multi-node cluster even when agent upgrades

>   were paused, caused by a missing or delayed run config at bootup that incorrectly forced an upgrade and introduced

Reviewer: ```suggestion
  were paused, caused by a missing or delayed run config at boot time that incorrectly forced an upgrade and introduced
```

<ReleaseNotesVersions />

## July 4, 2026 - Release 4.9.x

<!-- PATCH RELEASE TICKET: DOC-2957 -->

> ### Bug Fixes

Reviewer: ```suggestion
```

### Improvements

<!-- https://spectrocloud.atlassian.net/browse/PE-8884 -->

- The LocalTUI installation flow now detects Kairos partitions on all disks and blocks the install with a clear message
  identifying the affected disks, preventing unpredictable behavior from stale partitions.

<!-- https://spectrocloud.atlassian.net/browse/PE-8912 -->

> - Custom UI wizard bundles can now be packaged via palette-cli and rendered in Local UI on Slim ISO-bootstrapped edge

Reviewer 2: ```suggestion
- Custom UI wizard bundles can now be packaged via the [Palette CLI](../automation/palette-cli/palette-cli.md) and rendered in Local UI on Slim ISO-bootstrapped edge
```

  hosts, enabling content authors to ship tailored configuration experiences alongside their edge content.

### Bug Fixes

<!-- https://spectrocloud.atlassian.net/browse/PE-9004 -->

> - Fixed an issue where Stylus upgrades were triggered on nodes in a multi-node cluster even when agent upgrades were

Reviewer 2: Do we need to say "Palette Agent" rather than "Stylus" here?

> - Fixed an issue where Stylus upgrades were triggered on nodes in a multi-node cluster even when agent upgrades were

Reviewer: ```suggestion
- Fixed an issue where Palette Agent upgrades were triggered on nodes in a multi-node cluster even when agent upgrades were
```

<ReleaseNotesVersions />

> ## July 4, 2026 - Release 4.9.24

Reviewer 3: ```suggestion
## July 1, 2026 - Release 4.9.24
```

<!-- PATCH RELEASE TICKET: DOC-2957 -->

### Improvements

<!-- https://spectrocloud.atlassian.net/browse/PE-8884 -->

- While bootstrapping Edge hosts, the [TUI](/clusters/edge/site-deployment/site-installation/initial-setup) now checks
  all disks for partitions left behind by previous installations, preventing stale partitions from causing unpredictable
  installation behavior. Affected disks are flagged and pre-selected for wiping on the prerequisites screen; disks to

>   wipe must be confirmed on the following screen.

Reviewer 3: ```suggestion
- While bootstrapping Edge hosts, the [TUI](/clusters/edge/site-deployment/site-installation/initial-setup) now checks
  all disks for partitions left behind by previous installations, preventing stale partitions from causing unpredictable
  installation behavior. Affected disks are flagged and pre-selected for wiping on the prerequisites screen. Wiping disks is optional and must be confirmed on the following screen.
```

>   disks is optional and must be confirmed on the following screen.

Reviewer 2: ```suggestion
  disks is optional and must be confirmed on the follow-up screen.
```

Reviewer 2: Maybe? At the moment, it reads like a how-to.

<!-- https://spectrocloud.atlassian.net/browse/PE-8912 -->

- Content authors can now bundle a custom UI into Edge content using the `--custom-ui` flag of the

>   [Palette CLI](../automation/palette-cli/palette-cli.md) `content build` command. When the content is uploaded to a

Reviewer 3: ```suggestion
  [Palette CLI](/automation/palette-cli/commands/content/#build) `content build` command. When the content is uploaded to a
```
